### PR TITLE
Edit suggestions

### PR DESCRIPTION
**Pull request by raveling**

### CODE REVIEW

**Comment by bladey** [comment]
👍

File: docs/pages/docs/walkthroughs/getting-started-with-create-keystone-app.mdx [hunk 1]
```diff
@@ -56,72 +56,56 @@ npx create-keystone-app
...
 ## Output
 
-We now have some files in the newly generated folder. The two most important once you want to get into are `keystone.ts` and `schema.ts`.
+Keytone creates the following files in your newly generated folder. The most important ones are [`keystone.ts`](/docs/apis/config) and [`schema.ts`](/docs/apis/Schema).
 
 ```sh
 .
 ├── auth.ts        # Authentication configuration for Keystone
-├── keystone.ts    # The main entry file for Keystone
+├── keystone.ts    # Where you configure your project
```
Comment (Noviny) [inline docs/pages/docs/walkthroughs/getting-started-with-create-keystone-app.mdx:88]: I think the words 'entry file' are really helpful, but weak opinion

**Review by Noviny — COMMENTED** [review]
Just one tiny nit

File: docs/pages/docs/walkthroughs/getting-started-with-create-keystone-app.mdx (hunk 1 above)
Comment (raveling) [reply docs/pages/docs/walkthroughs/getting-started-with-create-keystone-app.mdx]: Fixed here:
https://github.com/keystonejs/keystone/pull/6586/commits/eba0328865531d11becd1d2bb58100ab1b0e236b

**Review by Noviny — APPROVED** [review]

**Pull request merged**